pyvista/core/filters/__init__.py

```diff
@@ -43,7 +43,7 @@ def _get_output(
     ido = algorithm.GetInputDataObject(iport, iconnection)
     data = wrap(algorithm.GetOutputDataObject(oport))
     if not isinstance(data, pyvista.MultiBlock):
-        data.copy_meta_from(ido)
+        data.copy_meta_from(ido, deep=False)
```

Comment:
Does this imply that putting a mesh through a filter will give us a new mesh that shares the active_* info and textures with the original? (Although most of this seems to be the _status quo_.)

Reply:
This is a mistake. Filters return a new dataset that's a "deep" copy of the original. Metadata should also be a deep copy without shared references.

Fixed in bd79d13748e278210ae53de1c3e2d365bcd6c087.